.gitignore

```diff
@@ -2,6 +2,9 @@
 __pycache__/
 *.py[cod]
 *$py.class
+library.db
+alembic.ini
+alembic
```

Comment:
Don't ignore alembic. It's like migrations folder that you must push so that via build.sh or docker_compose you run only migrate command, but makemigrations(creation of migration files before actual database changes) are run only on your local machine

Reply:
fixed